vagrant/forum/forum.py

```diff
@@ -40,68 +38,26 @@
 
 # HTML template for an individual comment
 POST = '''\
-    <div class=post><em class=date>%(time)s</em><br>%(content)s</div>
+    <div class=post><em class=date>%s</em><br>%s</div>
```

Comment:
Added to the lesson:

By the way, the warnings about using `%` with SQL statements and user data also apply to newer ways of doing string formatting in Python. This includes [f-strings, introduced in Python 3.6](https://docs.python.org/3/reference/lexical_analysis.html#f-strings).